tests/src/Context/FormContext.php

```diff
@@ -152,18 +152,4 @@ protected function fixStepArgument($argument) {
     return str_replace('\\"', '"', $argument);
   }
 
-  /**
-   * Check button doesn't exist.
-   *
-   * @Then I (should )not see the button :button
-   * @Then I (should )not see the :button button
-   */
-  public function assertNoButton($button) {
```

Comment:
Not used, remove instead of loosing time fixing.

Reply:
Question: we do you mean by "loosing time fixing"? 
The correct reason of its removing is because the step exists already in MinkContext and it is not buggy there, no?

Reply:
Duplicating a working step in MinkContext is alone a good reason to remove. We can put it together with not being used and not working.

Reply:
We are on the same line.
👍